### DISCUSSION TITLE
Drag & drop file

### DISCUSSION THREAD

**Discussion by nghia-vo** [zauberzeug/nicegui#2096] [Ideas / Feature Requests]
For a local app, I would like to enable users to drag and drop files into a designated area (similar to how users can drag and drop files to upload in Google Drive), then retrieve the file path and use it for further processing. Brief ideas as follow
```python
from nicegui import ui


def drop_file():
    """
    Do things here to get file path (local or server)
    """
    ui.notify('Your drop the file')


with ui.header().style("background-color: #3874c8").classes("items-center justify-between"):
    ui.label("Test")

with ui.left_drawer(fixed=True, bottom_corner=True).style("background-color: #d7e3f4"):
    with ui.card():
        drop_area = ui.label("Drop your file here").style('border: 2px dashed #ccc; padding: '
                                                          '40px; text-align: center;')
        drop_area.on("drop", drop_file)

ui.run()
```
How to achieve this?

**Comment by falkoschindler** [comment]
Is there a specific reason why you don't want to use `ui.upload`? It works as a file drop area.

**Reply by nghia-vo** [reply]
That's for server side. I would like to do it for local app where I can get a full path to data. Similar to what you did a demo for [local_file_picker](https://github.com/zauberzeug/nicegui/tree/main/examples/local_file_picker),

**Reply by falkoschindler** [reply]
I don't think it is possible to get the full path to a file on the client.
Have a look at this [discussion](https://github.com/zauberzeug/nicegui/discussions/283) for more background on the "local file picker".
